client/src/components/Search/SearchBox.scss

```diff
@@ -0,0 +1,192 @@
+$search-base-padding: 10px;
+$search-cancel-width: 52px;
+$search-filter-width: 35px;
+$search-enter-width: 52px;
+
+.search-box {
+  @include media-breakpoint-up(md) {
+    position: relative;
+    right: auto;
+
+    &.search-box__active {
+      right: 0;
+    }
+  }
+
+  &.search-box__active {
+    position: relative;
+    width: 100%;
+  }
+}
+
+.search-box__group {
+  opacity: 0;
+  overflow: hidden;
+  display: none;
+
+  .search-box__active & {
+    transition: opacity $transition-speed-fast;
+    overflow: visible;
+    opacity: 1;
+    display: block;
+    width: 100%;
+  }
+
+  .font-icon-search {
```

Comment:
You probably should chuck `pointer-events: none;` on this so it doesn't prevent clicking into the field.

Reply:
Done. I've applied to the _enter_ bit as well.